help/assets/css/styles.scss

```diff
@@ -0,0 +1,354 @@
+---
+---
+@import url('./fonts.css');
+
+:root {
+    --color-dark: #0B1B34;
```

Comment:
Can we create a separate file for all our colors, something like the following. Note that I took all the colors and theme variables from NewDot, but we could use a smaller set for HelpDot with only the colors and theme variables that we need.

```scss
$colors: {
    white: '#FFFFFF',
    gray1: '#FAFAFA',
    gray2: '#ECECEC',
    gray3: '#C6C9CA',
    gray4: '#7D8B8F',
    dark: '#0b1b34',
    black: '#000000',
    blue: '#0185ff',
    blueHover: '#0063bf',
    floralwhite: '#fffaf0',
    green: '#03d47c',
    greenHover: '#03c775',
    orange: '#ff7101',
    pink: '#f68dfe',
    red: '#fc3826',
    redHover: '#e13826',
    redDisabled: '#fea29a',
    yellow: '#fed607',
    transparent: 'transparent',
}

// Put the colors from our map into sass variables
:root {
    // each item in color map
    @each $name, $color in $colors {
        $color-#{$name}: #{$color};
    }
}
```

And then a separate file for each theme (i.e: just light for now):

```scss
@import url('./colors.css');

$appBackground: $color-white
$sidebar: $colors-gray1
...
...
```

The color variables are really good for the readability of the styles, while using a theme now will make it much easier to add darkmode and/or other themes later on.

Reply:
I tried to create the list of colors in this way but seems it's not possible. I tried using a [sass map](https://sass-lang.com/documentation/values/maps) but we can't create sass variables dynamically https://stackoverflow.com/a/8539474/1883585. I could give it a try again in https://github.com/Expensify/Expensify/issues/216742, what do you think?